src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,34 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.CountryDao;
 import mate.academy.hibernate.relations.model.Country;
 import mate.academy.hibernate.relations.service.CountryService;
+import mate.academy.hibernate.relations.util.DataProcessingException;
 
 public class CountryServiceImpl implements CountryService {
+    private CountryDao countryDao;
```

Comment:
It's recommended to make the `countryDao` field final to ensure it's not modified after construction of the `CountryServiceImpl` object.